src/CloudFoundry.VisualStudio/VsUtils.cs

```diff
@@ -1,25 +1,41 @@
-﻿using EnvDTE;
-using Microsoft.VisualStudio;
-using Microsoft.VisualStudio.ComponentModelHost;
-using Microsoft.VisualStudio.Shell.Interop;
-using NuGet.VisualStudio;
-using System;
-using System.Collections.Generic;
-using System.IO;
-using System.Linq;
-using System.Runtime.InteropServices;
-using System.Text;
-using System.Threading.Tasks;
-
-
-namespace CloudFoundry.VisualStudio
+﻿namespace CloudFoundry.VisualStudio
 {
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Runtime.InteropServices;
+    using System.Text;
+    using System.Threading.Tasks;
+    using EnvDTE;
+    using Microsoft.VisualStudio;
+    using Microsoft.VisualStudio.ComponentModelHost;
+    using Microsoft.VisualStudio.Shell.Interop;
+    using NuGet.VisualStudio;
+
     internal class VsUtils
     {
+        public static bool IsSelectedProjectWebsite
+        {
+            get
+            {
+                var project = VsUtils.GetSelectedProject();
+                if (project != null)
+                {
+                    return project.Object is VsWebSite.VSWebSite;
+                }
+                else
+                {
+                    return false;
+                }
+            }
+        }
+
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA1806:DoNotIgnoreMethodResults", MessageId = "Microsoft.VisualStudio.Shell.Interop.IVsAggregatableProject.GetAggregateProjectTypeGuids(System.String@)", Justification = "Result not releavant"), System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Usage", "CA1806:DoNotIgnoreMethodResults", MessageId = "Microsoft.VisualStudio.Shell.Interop.IVsSolution.GetProjectOfUniqueName(System.String,Microsoft.VisualStudio.Shell.Interop.IVsHierarchy@)", Justification = "Result not releavant")]
```

Comment:
Typo in releavant.
The results of GetAggregateProjectTypeGuids and GetProjectOfUniqueName should tell us if the operation s were successful though, so they should be checked.